docs/effects/lighting-effect.md

```diff
@@ -76,13 +17,13 @@ Parameters:
 
 ##### `ambientLight` (Object, optional)
 
-An ambient light source which illuminates all the objects equally.
+An [AmbientLight](/docs/api-reference/lights/ambient-light.md) source which illuminates all the objects equally.
 
 * Default: ambient light source with color = `[255, 255, 255]` and intensity = `1.0`
 
 ##### `directionalLights` (Array, optional)
 
-Array of Directional light source which emits from a specific directions.
+Array of [DirectionalLight](/docs/api-reference/lights/directional-light.md) source which emits from a specific directions.
```

Comment:
These are not props since these keys and values cannot be passed to the constructor.